docs/modules/jenkins-slaves/pages/maven.adoc

```diff
@@ -0,0 +1,27 @@
+= Maven / Gradle Slave
+
+This slave is used to build java code, both thru `maven` and `gradle`
+
+== Features
+
+* Nexus configuration for `Maven` and `Gradle`
+* HTTP Proxy awareness for `Maven` and `Gradle`
+* Default Installation maven 3.3 based on https://www.softwarecollections.org/en/scls/rhscl/rh-maven33/[rh-maven33]
+* Default Gradle Installation 5.6.1
```

Comment:
I think it is way older?

Reply:
you are right. I upgraded it for our installation. will provide PR for that one.